src/hooks/useLoadInitialData.tsx

```diff
@@ -133,4 +136,42 @@ export const useLoadInitialData = () => {
         const unsub = onSyncStatus(() => updateUiDriveNodes(documentDrives));
         return unsub;
     }, [documentDrives, onSyncStatus, updateUiDriveNodes]);
+
+    // Updates the initial loading status for drives when the initial sync is done.
+    // if the syncSatatus is different from SYNCING (meaning the sync either failed or succeeded)
+    // we set initialSyncStatus for the drive as 'READY'
+    // initialSyncStatus === 'READY' means the initial sync for the drive has been finished (either failed or succeeded)
+    useEffect(() => {
+        const initialSyncRequired = initialLoadingStatus.some(
+            drive => drive.initialSyncStatus !== 'READY',
+        );
+
+        if (!initialSyncRequired) return;
+
+        const unsub = onSyncStatus((driveId, status) => {
```

Comment:
Here we update the initial sync status of the drive to READY when the new SyncStatus is not equal to 'SYNCING' (indicating that the sync has either finished successfully or failed).